modules/setting-up-cpu-manager.adoc

```diff
@@ -52,9 +52,9 @@ spec:
 ----
 <1> Specify a policy:
 * `none`. This policy explicitly enables the existing default CPU affinity scheme, providing no affinity beyond what the scheduler does automatically.
-* `static`. This policy allows pods with certain resource characteristics to be granted increased CPU affinity and exclusivity on the node.
+* `Static`. This policy allows pods with certain resource characteristics to be granted increased CPU affinity and exclusivity on the node. If `static`, you must use a lowercase `s`.
```

Comment:
should this remain 
> * `static`.

as originally shown, but with the last statement included?